iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.csproj

```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.8.0" />
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.33.0" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.33.1" />
```

Comment:
this version of the sdk gives us the correct status callback on initial disconnect (so we don't need the `wasEverConnceted` etc. flag anymore).